### ISSUE TITLE
Photos uploaded in public post get limited permissions

### ISSUE THREAD

**Issue by SpcCw** [friendica/friendica#8335]
### Expected behavior
If post is public, images uploaded during creation of the post (via "add image") should also get public permission.

### Actual behavior
Images are uploaded with "limited/private" permission and no one in the access list.

### Steps to reproduce the problem
Have default permission setting for new posts as "public". Create new post (public). Add some images by using "Image" button, selecting an album and then uploading images from this dialog. Go to photo properties and it is "limited".

Or post it and get comments with people complaining :)

### Friendica version you encountered the problem
2019.12

### Friendica source (git, zip)
git
### PHP version
PHP 7.4.2

Possibly related to #5805 - looks like exact same issue?

**Comment by MrPetovan** [comment]
Is this fixed with https://github.com/friendica/friendica/pull/8384?

**Comment by SpcCw** [comment]
Okay... you all are going to hate me now... it worked all along! I did some more testing (not on my node even) and discovered the problem. 

One thing I dislike in Friendica is how it presents images. By default it embeds large images as `...0905e67c4a42ff86915209581-1.jpg` links which limits width to 640px. Which is great for your Friendica contacts but bad for every other network as the image is small and they can't easily zoom in. I discovered that inserting the image and replacing -1.jpg with -0.jpg or just deleting -1 part completely produced full sized image - so if I resized picture before uploading, it would be reasonably lean and yet fill as much as formatting on the other end let. Plus it gave opportunities to use right click zoom and such.

Today I found that deleting this suffix after uploading image via post creation dialog screwed up permissions. On my node there are none set and on RC the only permission is "Yourself". However if URL is changed from -1 to -0, it works as expected. On the older version too.

And of course because I sometimes deleted suffix, sometimes replaced it with -0 it worked intermittently. Stupid me! Closing this.
